test/torch/tensors/test_native.py

```diff
@@ -117,15 +117,15 @@ def test_copy():
     assert tensor is not coppied_tensor
 
 
-def test_size():
+def test_local_size():
```

Comment:
why were these tests renamed?

Reply:
This is just some leftover from #2343

Reply:
Would you mind undoing the changes in this file if possible? so we can keep the PR as simple and narrow as possible

Reply:
Yeah, I'm going to do that tomorrow morning for sure